lib/commands/new.ts

```diff
@@ -90,6 +96,22 @@ command = {
 		// TODO: update output path based on where the CLI is called
 		await projTemplate.generateFiles(process.cwd(), argv.name, theme);
 		Util.log("Project Created");
+
+		this.git(argv["skip-git"], argv.name);
+	},
+	git(skipGit, projectName) {
+		if (!skipGit) {
+			process.chdir(projectName);
```

Comment:
Can this be passed directly to [exec `cwd` option](https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options)? Will also skip going back afterwards.